Makefile

```diff
@@ -93,6 +105,7 @@ remote-cluster-deploy:
 	clusterctl generate cluster $(TEST_CLUSTER_NAME) \
 		--kubernetes-version $(K8S_VERSION) \
 		--infrastructure linode-linode:$(CAPL_VERSION) \
+		--control-plane-machine-count 1 --worker-machine-count 1 \
```

Comment:
Question: Any reason to create 2 nodes for testing? Is one node getting overload?

Reply:
Pods were not getting scheduled on control-plane node due to taints. Hence, I provisioned a worker node so that pods can be scheduled there by the tests.